app/main.js

```diff
@@ -41,7 +43,10 @@ global.shared = {
 const gotTheLock = app.requestSingleInstanceLock()
 
 if (!gotTheLock) {
-  console.log('Stretchly is already running.')
+  console.log('Stretchly is already running, sending command\n')
```

Comment:
I've recently added `electron-log` package for logging. I think we could try using it for logging this as well. 

I've been using this way of logging: https://github.com/hovancik/stretchly#logging Maybe use something like `Stretchly 2nd instance: message` or sth like that. I guess it should go to same file.

Reply:
See https://github.com/papey/stretchly/commit/554a61293032a69859973f50b6e5c52be373b4b2

Reply:
Awesome :) If you see some place that you think could be good one for some debug message, feel free to add it :)